pubspec.yaml

```diff
@@ -28,6 +28,7 @@ dependencies:
   google_fonts: ^0.5.0+1
   # Picking images from the image library
   image_picker: ^0.6.6+5
+  transparent_image: ^1.0.0
```

Comment:
I think it provides redudant functionality. I suggest you try the flutter's built in fadeinimg widget.  
https://api.flutter.dev/flutter/widgets/FadeInImage-class.html
```
      return FadeInImage(
        placeholder: AssetImage("assets/images/icon_ecell.png"),
        image: NetworkImage(this.profilePicUrl),
      );
```